recipe/meta.yaml

```diff
@@ -1,111 +1,214 @@
+    - ninja
+    - ruby                               # [linux]
+    - bison                              # [linux]
+    - flex                               # [linux]
+    - gperf                              # [linux]
+    - m2-gperf                           # [win]
+    - m2-bison                           # [win]
+    - m2-flex                            # [win]
+    - m2-patch                           # [win]
+    - jom                                # [win]
+    - perl 5.*
   host:
-    - 7za  # [win]
-    - bison =3.0.4  # [unix]
-    - flex  # [unix]
-    - gperf  # [unix]
-    - msinttypes  # [win and py27]
-    - freetype  # [linux]
-    - fontconfig  # [linux]
-    - openssl  # [not osx]
-    - jpeg
-    - zlib
-    - libpng
-    - gstreamer  # [linux]
+    - dbus                               # [linux]
+    - fontconfig                         # [linux]
+    - freetype                           # [linux]
+    - gst-plugins-base                   # [linux]
+    - gstreamer                          # [linux]
+    - glib                               # [linux]
+    - libxml2                            # [linux]
+    # For WebEngine, but for compatiblity we want to restrict
+    # usage of our own libraries. It seems Chromium prefers
+    # expat to libxml2 and we will try to froce it to use the
+    # bundled one.
+    # Other than that (and esp. for Google developed libs) we
+    # avoid our own packagess. FFMPEG was not detected anyway,
+    # and neither was libvpx (AFAICT).
+    # - libxml2                            # [unix]
+    # - libxslt                            # [unix]
+    # - libopus                            # [unix]
+    # - libvpx                             # [unix]
+    # - ffmpeg                             # [unix]
+    - expat                              # [linux]
     - icu
-    - sqlite
-    - libxcb  # [linux]
-    - dbus  # [linux]
-    - xorg-libice      # [linux]
-    - xorg-libsm       # [linux]
-    - xorg-libx11      # [linux]
-    - xorg-libxext     # [linux]
-    - xorg-libxrender  # [linux]
-    - xorg-libxt       # [linux]
-    - xorg-inputproto  # [linux]
-    - gtk2  # [linux]
-  run:
-    - freetype  # [linux]
-    - fontconfig  # [linux]
-    - openssl  # [not osx]
     - jpeg
-    - zlib
     - libpng
-    - gstreamer  # [linux]
-    - icu
     - sqlite
-    - libxcb  # [linux]
-    - dbus  # [linux]
-    - xorg-libice      # [linux]
-    - xorg-libsm       # [linux]
-    - xorg-libx11      # [linux]
-    - xorg-libxext     # [linux]
-    - xorg-libxrender  # [linux]
-    - xorg-libxt       # [linux]
-    - gtk2  # [linux]
+    - zlib
+    - libxcb                             # [linux]
+    # We cannot install OpenSSL into the prefix when building qtwebengine
+    # on linux as BoringSSL and OpenSSL then conflict with each other.
+    - openssl 1.1.*                      # [win]
+  run:
+    - openssl 1.1.*                      # [not osx]
+    - libxcb                             # [linux]
```

Comment:
@mingwandroid, why ICU is not part of the run dependencies here? That's what's making the tests fail on macOS.